### PR TITLE
Adds Dollar Shave Club, Elsevier, and Trivago resources

### PR DESCRIPTION
**Pull request by thomgoodman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,6 +224,15 @@ Happy Learning!
 
 </details>
 
+<details>
+  <summary>Dollar Shave Club</summary>
+
+#### Blogs
+* [How Dollar Shave Club 3x'd velocity and learned to love their tests](https://engineering.dollarshaveclub.com/qa-environments-on-demand-with-kubernetes-5a571b4e273c)
```
Comment (abhivaikar) [inline README.md:231]: Looks like the URLs are interchanged. Can you change that? Thanks!
```suggestion
* [How Dollar Shave Club 3x'd velocity and learned to love their tests](https://circleci.com/blog/how-dollar-shave-club-3x-d-velocity-and-learned-love-tests)
```

File: README.md [hunk 2]
```diff
@@ -224,6 +224,15 @@ Happy Learning!
...
+* [QA Environments on Demand with Kubernetes](https://circleci.com/blog/how-dollar-shave-club-3x-d-velocity-and-learned-love-tests)
```
Comment (abhivaikar) [inline README.md:232]:
```suggestion
* [QA Environments on Demand with Kubernetes](https://engineering.dollarshaveclub.com/qa-environments-on-demand-with-kubernetes-5a571b4e273c)
```

**Review by abhivaikar — CHANGES_REQUESTED** [review]

**Comment by abhivaikar** [comment]
Thanks a lot for this PR @thomgoodman! 👏 The quality engineering at Elsevier article is really interesting!

**Comment by thomgoodman** [comment]
Good catch, fixed. Thanks for creating this great collection @abhivaikar !

**Review by abhivaikar — APPROVED** [review]

**Pull request merged**